tools/internal_ci/linux/grpc_psm_performance_gke_experiment.sh

```diff
@@ -100,6 +100,7 @@ psmBuildConfigs() {
         -a ci_gitCommit="${GRPC_GITREF}" \
         -a ci_gitCommit_java="${GRPC_JAVA_GITREF}" \
         -a ci_gitActualCommit="${KOKORO_GIT_COMMIT}" \
+        -a enablePrometheus="true" \
```

Comment:
true does not need quotation marks in this case.

Reply:
Right, with or without the quotation it produce the same results in test-infra. Let me remove it.